src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -411,6 +412,20 @@ protected override int MakeSlot(BoundExpression node)
         {
             switch (node.Kind)
             {
+                case BoundKind.Conversion:
+                    {
+                        var conv = (BoundConversion)node;
+                        var operand = conv.Operand;
+                        if (conv.Conversion.Kind == ConversionKind.ExplicitNullable &&
+                            operand.Type?.IsNullableType() == true &&
+                            conv.Type?.IsNullableType() == false)
+                        {
+                            // Explicit conversion of Nullable<T> to T is equivalent to Nullable<T>.Value.
+                            int containingSlot = MakeSlot(operand);
+                            return containingSlot < 0 ? -1 : GetNullableOfTValueSlot(operand.Type, containingSlot);
```

Comment:
> return containingSlot < 0 ? -1 : GetNullableOfTValueSlot(operand.Type, containingSlot); [](start = 27, length = 88)

I would like to better understand why do we need to be able to get a slots for conversions. Could you provide some details? Offline would be fine.